packages/contentstack-export/src/export/modules-js/marketplace-apps.js

```diff
@@ -44,7 +44,10 @@ module.exports = class ExportMarketplaceApps {
     }
 
     this.developerHubBaseUrl = this.config.developerHubBaseUrl || (await getDeveloperHubUrl(this.config));
-
+    this.appSdkAxiosInstance = await managementSDKClient({
+      host: this.developerHubBaseUrl.split('://').pop(),
+      endpoint: this.developerHubBaseUrl,
```

Comment:
@cs-raj Can you confirm for me, should we need to pass both the host and endpoint.? Either one of them is enough right.?

Reply:
Done @antonyagustine used the host only